.github/workflows/continuous-release.yml

```
@@ -0,0 +1,24 @@
+name: Publish Every Commit
+on: [push, pull_request]
```

This seems a bit excessive. It will run twice for every commit in a PR and once for every other commit. I think this is what you actually want:

```suggestion
name: Publish Every Commit
on:
  push:
    branches:
      - main
  pull_request:
    types:
      - opened
      - synchronize
```